compatibility.bs

```diff
@@ -1055,11 +1057,61 @@ On mobile platforms, including smaller iPad form factors
   </tbody>
 </table>
 
+
+<h2 id="removal">Deprecation and Removal</h2>
+
+<em>This section is non-normative.</em>
+
+While the web platform stives for backwards-compatibility, there are occasions which require the
+outright removal or discouraged use of some features. To support making these changes
+in the least disruptive and most consistent way for developers, we document them here. This is most
+beneficial as a decalration of intent by browser engines to act together in a well reasoned way.
+
+<h3 id="removal-features">Feature List</h3>
```

Comment:
There are things that user agents can do that are platform impacting without actual standardization changes; would that information be expected to join this list as well? Many of the privacy controls come to mind and it's not that there aren't available solutions to be leveraged in many of these contexts but it would help centralize potential customer impacting scenarios.

Reply:
I agree with @bvandersloot-mozilla! I would add that while I generally agree that it would be useful for changes of many sorts to end up in a list like this one, it's likely going to continue being the case that some kinds of changes show up elsewhere _first_ given the various requirements and audiences that browser vendors need to satisfy.

Reply:
Thank you @bvandersloot-mozilla, sounds good. I think it would be good and try people default to informing while I think we all understand that there may be some things that don't make its way into this doc for a variety of reasons but I'd prefer for that to be the exception and not the rule. That doesn't require any text changes but strengthening the culture around this when this lands so please consider my comment resolved :)